### PR TITLE
adding declarative to Elm Tags.md (see comment link)

### PR DESCRIPTION
**Pull request by gacallea**
as mentioned here, functional languages should be tagged declarative too. https://github.com/exercism/haskell/pull/1124#issuecomment-1312702666

### CODE REVIEW

**Comment by ceddlyburge** [comment]
Hi Andrea, I have no strong opinion about this, but I think that others may do. If nobody replies after a while then I'll approve it.
Thanks!
Cedd

**Comment by gacallea** [comment]
Hi @ceddlyburge :)

tbh, I would not add declarative to any functional language. However, if Haskell has to have it I'd add it here (and more?) for consistency. Not meaning to criticize here, just trying to help with the languages I care the most about. That said, Exercism should have an official policy/rule about this. Perhaps it's my OCD talking, don't mind me :P

**Comment by ceddlyburge** [comment]
Good to know! I don't really mind either way, and it does make sense to be consistent within exercism. I'll give Matthieu and Jeremie a couple of days to wade in with an opinion if they want to, and if they don't I'll merge it.

**Comment by ceddlyburge** [comment]
Funny how emotive these details get sometimes :)

**Review by jiegillet — APPROVED** [review]
I'm good with it, from what I can read about declarative programming languages, Elm is a great fit for it.

**Comment by mpizenberg** [comment]
Yep, good with it too

**Comment by gacallea** [comment]
@ceddlyburge @jiegillet @mpizenberg I've added TAGS.md as well, since it was missing. Please double check that before merging. Sorry, I just noticed.

File: TAGS.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- [ ] `runtime/wasmtime`: no support.
+
+## Used for
+
+- [ ] `used_for/artificial_intelligence`: N/A
+- [ ] `used_for/backends`: N/A
+- [ ] `used_for/cross_platform_development`: N/A
+- [ ] `used_for/embedded_systems`: N/A
+- [ ] `used_for/financial_systems`: N/A
+- [x] `used_for/frontends`: Elm only focus is frontend development.
```
Comment (ceddlyburge) [inline TAGS.md:51]:
```suggestion
- [x] `used_for/frontends`: Elm's only focus is frontend development.
```

**Review by ceddlyburge — APPROVED** [review]

**Comment by gacallea** [comment]
yay! thank you all :)

**Pull request merged**